### PR TITLE
Liam Barstad

### PR DESCRIPTION
**Pull request by liambarstad**

### CODE REVIEW

File: source/addComment.js [hunk 1]
```diff
@@ -1,3 +1,22 @@
 $(document).ready(function(){
-  // add your code here
+  $('#comments #new_comment_button').on('click', function(e) {
+    e.preventDefault();
+    var newCommentButton = $('#comments #new_comment_button');
+    newCommentButton.detach();
```
Comment (kat3kasper) [inline source/addComment.js:5]: I haven't seen the `detach()` method before. Cool usage. I think it may have been more useful in the line before though.

File: source/addComment.js [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+    e.preventDefault();
+    var newCommentButton = $('#comments #new_comment_button');
+    newCommentButton.detach(); 
+    $('#comments #comment_list').append('<li id="new_comment">\n</li>');
+    $('#comment_list #new_comment').append('<form></form>');
+    $('#new_comment form').append('<input type="text" placeholder="Comment" id="new_comment_body">');
+    $('#new_comment form').append('<input type="text" placeholder="Author" id="new_comment_author">');
+    $('#new_comment form').append('<input type="submit" value="Submit" id="new_comment_submit">');
+    document.getElementById('new_comment_submit').addEventListener('click', function(e) {
+      e.preventDefault();
```
Comment (kat3kasper) [inline source/addComment.js:12]: You shouldn't have to `preventDefault` here. It is useful on submit as you don't want the page to reload but it doesn't help us with `click` events

**Review by kat3kasper — COMMENTED** [review]
I would have loved to see some of the code broken up into functions. i.e. `addFormToPage`, `addCommentToPage` This breaks up functionality leaving the code more readable.

**Comment by kat3kasper** [comment]
@liambarstad
